### PR TITLE
Added 3 new fields to Call out box paragraph

### PR DESCRIPTION
**Pull request by gosia-mlynarczyk**

### CODE REVIEW

**Review by Adnan-cds — COMMENTED** [review]
I was testing this in a fresh Localgov Drupal install and the new title field and external link feature were not working.  Turned out the localgov_theme has overridden the template for the Call out box paragraph template.  This made the theme template changes from this pull request of no use.  Once I dropped the relevant file from the localgov_theme, all worked.

Given that we haven't been into this situation before, do we need an accompanying pull request for the localgov_theme before accepting this in?  Or is it okay to accept this given that the changes from this pull request look okay?

FAO @finnlewis @cjstevens78 @gosia-mlynarczyk

File: modules/localgov_campaigns_paragraphs/templates/paragraph--localgov-call-out-box.html.twig [hunk 1]
```diff
@@ -32,18 +32,32 @@
...
  * @see template_preprocess_paragraph()
  *
  * @ingroup themeable
  */
 #}
-<div class="call-out-box" {% if background_url is defined and background_url %}style="background-image: url('{{ background_url }}')"{% endif %}>
+{%
+  set classes = [
+    'call-out-box',
+    colour_theme ? 'call-out-box--' ~ colour_theme|clean_class : ''
```
[the remark this answers is not in the document]
Comment (gosia-mlynarczyk) [reply modules/localgov_campaigns_paragraphs/templates/paragraph--localgov-call-out-box.html.twig]: You're right Adnan, changed

**Comment by gosia-mlynarczyk** [comment]
Ah yes, I haven't done `git pull` in the `localgov_theme` since I've started, so it was working for me. 

Should I merge this in, and then create a separate pull request in the `localgov_theme` repo to update the theme template?
What do you think @Adnan-cds @finnlewis @cjstevens78 ?

**Comment by cjstevens78** [comment]
> Ah yes, I haven't done `git pull` in the `localgov_theme` since I've started, so it was working for me.
> 
> Should I merge this in, and then create a separate pull request in the `localgov_theme` repo to update the theme template?
> What do you think @Adnan-cds @finnlewis @cjstevens78 ?

@gosia-mlynarczyk yes i think this is the best approach, if you mark me as a reviewer on that ill make sure everything's ok on the theme side. I recently made some changes to the call out boxes mainly around how the images are displayed - which required the creation of the theme template file.

**Comment by Adnan-cds** [comment]
> Should I merge this in, and then create a separate pull request in the localgov_theme repo to update the theme template?

That would be very kind of you.

Please note that all the automated test runs are failing at the moment.  This is possibly due to a new release of phpstan/phpstan.  Changes from this pull request shouldn't break any test, so this one is okay to merge.

**Pull request merged**